f2/src/components/formik/radio/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
Can we merge Radio and Checkbox into one component and pass in a type prop? Those two components are mostly the same except the type.

Reply:
This was discussed with @ngosset and it was decided that these should remain separate in case different functionality is required in the future.